plotsScript.py

```diff
@@ -8,43 +8,53 @@
 from scipy.interpolate import interp1d
 from scipy.integrate import cumtrapz
 import pyfits
+import pdb
+
 from sklearn.grid_search import GridSearchCV
 from sklearn.svm import SVC, LinearSVC
 from sklearn.neighbors.kde import KernelDensity
 from astroML.plotting.tools import draw_ellipse
 
+import lsst.afw.table as afwTable
+import lsst.pex.exceptions
+
 import supervisedEtl as etl
 import dGauss
+import utils
+import sgSVM as sgsvm
```

Comment:
utils is used in the function magExtPlots.
sgSVM is unused and I'm removing it.